gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Mar 23 17:01:59 EET 2020
+#Wed Oct 21 17:56:49 EET 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```

Comment:
gradle distributions packaged with --all should be preferred instead of bin.
check [this](https://docs.gradle.org/current/userguide/gradle_wrapper.html) for more info about gradle wrapper